### PR TITLE
maven command change

### PR DESCRIPTION
**Pull request by Sreejith-Websphere**
#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] IBM Equal Access Accessibility Checker (https://www.ibm.com/able/toolkit/verify/automated)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

File: src/main/content/_assets/js/builds.js [hunk 1]
```diff
@@ -1006,7 +1006,7 @@ $(document).ready(function () {
                     setFocusOnModalPopupElement();
                     var build_tool = $('input[name=\'build_system\']:checked').val();
                     if(build_tool == "maven") {
-                        $('#cmd_to_run').text("mvnw liberty:run");
+                        $('#cmd_to_run').text("mvnw liberty:dev");
                     }
                     else {
                         $('#cmd_to_run').text("gradlew libertyStart");
```
Comment (kinueng) [inline src/main/content/_assets/js/builds.js]: the Gradle command also needs to be updated to `gradlew libertyDev` to match `mvnw liberty:dev`

**Review by kinueng — CHANGES_REQUESTED** [review]

File: src/main/content/_assets/js/builds.js (hunk 1 above)
Comment (kinueng) [reply src/main/content/_assets/js/builds.js]: Additionally, we need to update the README to also use `gradlew libertyDev` and `mvnw liberty:dev`
See https://github.com/OpenLiberty/start.openliberty.io/blob/main/src/main/resources/templates/readme/README.txt

Comment (Sreejith-Websphere) [reply src/main/content/_assets/js/builds.js]: ok

**Comment by Sreejith-Websphere** [comment]
command is updated in openliberty starter readme file https://github.com/OpenLiberty/start.openliberty.io/pull/79

**Review by kinueng — APPROVED** [review]

**Pull request merged**